ui/app/clinical/displaycontrols/allvisits/views/visitsTable.html

```diff
@@ -14,7 +14,7 @@ <h2 class="section-title" ng-show="::params.translationKey" >
                 <td class="name" width="50%">
                     <a class="visit" ng-click="openVisit(visit)">
                         <span>{{::visit.startDatetime | bahmniDate}}</span>
-                        <span ng-if="::!visit.isActive()">- {{::visit.stopDatetime | bahmniDate }}</span>
+                        <span ng-if="::(!visit.isActive() && (visit.stopDatetime | bahmniDate) !== (visit.startDatetime | bahmniDate))"> - {{::visit.stopDatetime }}</span>
```

Comment:
@ShwetaKamboj-TW Seems like you have missed the bahmniDate filter here. So the date format displayed will be different for stopDate.